dictionary/.eslintrc.json

```diff
@@ -1,5 +1,8 @@
 {
   "parserOptions": {
     "sourceType": "script"
+  },
+  "rules":{
+    "linebreak-style": 0
```

Comment:
I think this is a very important rule. It makes sure that people don't accidentally use CRLF.

Can we fix the Windows problem with other methods?

Reply:
tried another approach

Reply:
I have removed lint from the test command and I check linting only on one os/node version (useless to do linting on all platforms/node version)